files/build_templates/docker_image_ctl.j2

```diff
@@ -180,7 +183,24 @@ start() {
 
     if [ -z "$DEV" ]; then
         NET="host"
-    else
+
+        # For Multi-ASIC platform we have to mount the redis paths for the DB's running in different
+        # namespaces, which is of this format /var/run/redis<0,1,2,...>, into the single instance
+        # dockers like snmp, pmon on linux host.
+        {%- if docker_container_name != "database" %}
+        if [ $NUM_ASIC -gt 1 ]; then
+            asic_inst=0;
+            while [ $asic_inst -lt $NUM_ASIC ]; do
+                REDIS_MNT=$REDIS_MNT" -v /var/run/redis$asic_inst:/var/run/redis$asic_inst:rw "
```

Comment:
>/var/run/redis$asic_inst [](start = 66, length = 24)

The ground truth is from database_config, not to concat here.

Reply:
Sure, I have updated the code to take the redis_dir's to be mounted from the database_global.json file.